> Performing static analysis using mypy
pandas/plotting/_core.py:1763: error: Item "None" of "Optional[Tuple[EntryPoint, ...]]" has no attribute "__iter__" (not iterable)  [union-attr]
pandas/plotting/_core.py:1766: error: Invalid self argument "EntryPoint" to attribute function "load" with type "Callable[[EntryPoint], Any]"  [misc]
Found 2 errors in 1 file (checked 1297 source files)
Performing static analysis using mypy DONE

--- a/pandas/plotting/_core.py
+++ b/pandas/plotting/_core.py
@@ -1760,11 +1760,12 @@ def _load_backend(backend: str) -> types.ModuleType:
     found_backend = False
 
     eps = entry_points()
-    for entry_point in eps.get("pandas_plotting_backends"):
-        found_backend = entry_point.name == backend
-        if found_backend:
-            module = entry_point.load()
-            break
+    if "pandas_plotting_backends" in eps:
+        for entry_point in eps["pandas_plotting_backends"]:
+            found_backend = entry_point.name == backend
+            if found_backend:
+                module = entry_point.load()
+                break
 
     if not found_backend:
         # Fall back to unregistered, module name approach.

